iceberg/iceberg-handler/src/test/queries/positive/vectorized_iceberg_read_orc.q

```diff
@@ -20,14 +22,17 @@ create external table tbl_ice_orc_all_types (
     t_date DATE,
     t_decimal DECIMAL(4,2)
     ) stored by iceberg stored as orc;
-
-insert into tbl_ice_orc_all_types values (1.1, 1.2, false, 4, 567890123456789, '6', "col7", cast('2012-10-03 19:58:08' as timestamp), date('1234-09-09'), cast('10.01' as decimal(4,2)));
+--casting throws ClassCastException, Disable untill fixed.
```

Comment:
if it's a known issue, please create JIRA and link it here

Reply:
No longer an issue now, Sorted in this PR. :-)